sql/pgwire_test.go

```diff
@@ -291,3 +294,58 @@ func TestPGPrepared(t *testing.T) {
 		}
 	}
 }
+
+func startInsecureServer(t *testing.T) (addr url.URL, done func()) {
+	ctx := server.NewTestContext()
+	ctx.Insecure = true
+	s := setupTestServerWithContext(t, ctx)
+
+	pgUrl := url.URL{
+		Scheme:   "postgres",
+		Host:     s.PGAddr(),
+		RawQuery: "sslmode=disable",
+	}
+
+	return pgUrl, func() {
+		cleanupTestServer(s)
+	}
+}
+
+func TestPGPython(t *testing.T) {
+	defer leaktest.AfterTest(t)
+
+	pgUrl, cleanupFn := startInsecureServer(t)
+	defer cleanupFn()
+
+	host, port, err := net.SplitHostPort(pgUrl.Host)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	c := exec.Command("docker",
```

Comment:
take a look at `./acceptance/run.sh` for setting up Docker. There's also an argument that this sort of testing should move into the `acceptance` package.